.github/workflows/release-beta.yaml

```
@@ -0,0 +1,39 @@
+      
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: latest
+          
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+      - name: Install dependencies
+        run: |
+          pnpm install
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)